storm-core/src/clj/org/apache/storm/daemon/supervisor.clj

```diff
@@ -269,10 +269,7 @@
   (catch IOException e
     (log-warn-error e "Failed to cleanup worker " id ". Will retry later"))
   (catch RuntimeException e
-    (log-warn-error e "Failed to cleanup worker " id ". Will retry later")
-    )
-  (catch java.io.FileNotFoundException e (log-message (.getMessage e)))
```

Comment:
Delete redundant catch of FileNotFoundException since it has already been caught as IOException